.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+.[0-9]+'
+  pull_request: ~
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: "ubuntu-20.04"
+    timeout-minutes: 10
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php:
+          - '7.1'
+          - '7.3'
```

Comment:
Out of curiosity, why are we not testing against `7.2`?

If were doing it to save time, then it makes sense to not test against `7.3` as well, since using syntax structures not available in it would already be covered by `7.1`.

Reply:
While it's true that 7.1 and 7.2 are reaching end of their lifetime, our product declares itself as compatible with those. We can't drop support for them in older versions because it would be a Backward Compatibility break. Removing support for PHP engine is prohibited in semver convention, which we follow.